src/components/galleryCard/GalleryCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Card, CardTitle, CardImage, CardContent, CardFooter } from '../card'
+import { Tag } from '../../components/tag'
+import { H5, H6, P } from '../typography'
+import { colors } from '../../theme/colors'
+import { GalleryCardProps } from './GalleryCard.types'
+
+const StyledTag = styled(Tag)`
+  display: flex;
+  transform: translate(5px, -35px);
```

Comment:
This might be easier with a `position: relative` and then moving it upwards with a `top: Npx`, rather than transforming on both axes.

Reply:
Also, this `Tag` should only be the length of the content - not the width of the `GalleryCard` component.